study-builder/tenants/testboston/pages/login.html

```diff
@@ -156,6 +156,31 @@
         showPasswordRecentText = true;
     }
 
+    if (mode === 'signup') {
+        realOptions.additionalSignUpFields = [
+            {
+                name: "first_name",
+                placeholder: "Your first name",
+                validator: function(input) {
+                  return {
+                     valid: input != null && input.length > 0,
+                     hint: "Can't be blank"
+                  };
+                }
+            },
+            {
+                name: "last_name",
+                placeholder: "Your last name",
```

Comment:
Does auth0 provide way to internationalize these (a question. Not asking you do it).

Reply:
On a cursory look, [here](https://auth0.com/docs/universal-login/i18n#internationalization-in-the-classic-universal-login-experience) is what Auth0 says about i18n. We're using their old/classic universal login, so sounds like they have built-in i18n for their Lock widget. However, I couldn't find mentions of what they do for custom fields. Since `validator` is just a function, I imagine we can roll our own -- just two fields for now -- using our own dictionary (pseudocode):
```js
var dict = {
  en: { blank_hint: 'can't be blank!' },
  es: { blank_hint: '...' }
};
...
validator: function(input) {
  return { ... hint: dict[language].blank_hint ... };
}
```

Reply:
I looked at this and this definitely seems like the approach that I was considering for the custom field internationalization.  One thing I haven't looked into much yet is the non-custom fields.  For some of those fields (and some elements we specifically add), Prion uses the language dictionary to specify values other than the default.  My sense is that Auth0 wouldn't be able to translate those, so we might do something similar for those.  For fields where we are using the Auth0 default, it might still be worth doing something like this so we know we have consistent translation.  I believe Auth0 specifies a set of available languages, so it's also possible that we could have a situation where we are using a language that Auth0 doesn't translate things into.  So certainly at least some of the time we will need to use an approach like this, and it is possible we might want to do this all of the time.